.ci/pipeline/release_jjb.yaml

```diff
@@ -0,0 +1,65 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: pipeline
+    folder: libvma
+    properties:
+        - github:
+             url: "https://github.com/Mellanox/libvma"
+        - build-discarder:
+            days-to-keep: 50
+            num-to-keep: 20
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+    description: Do NOT edit this job through the Web GUI !
+    concurrent: false
+    parameters:
+        - string:
+            name: "release_tag"
+            default: ""
+            description: "Tag to release"
+        - string:
+            name: sha1
+            default: 'tags/$release_tag'
+            description: 'commit to use, defaults to the commit tag provided under release_tag'
+        - string:
+            name: "revision"
+            default: "0"
```

Comment:
should be 1 not 0

Reply:
This is whats currently under the vma release folder (all revision numbers are set to 0)
We need to check if they want it changed to starting with 1 instead

Reply:
So fix the default revision in the script, they must match